modules/ding_dams/css/ding_dams.media_edit.css

```diff
@@ -0,0 +1,68 @@
+/* @TODO: Fix !important styles, cause they are a fast hack for it to work. */
+div.ctools-modal-content {
+  /* @TODO: Disallow !important - Use of !important. */
+  width: 710px !important;
+  /* @TODO: Disallow !important - Use of !important. */
+  height: 330px !important;
+}
+div#modal-content {
+  display:block;
+  /* @TODO: Disallow !important - Use of !important. */
+  width: 700px !important;
+  /* @TODO: Disallow !important - Use of !important. */
+  height: 330px !important;
+  /* @TODO: Disallow !important - Use of !important. */
+  margin: 0 !important;
+  /* @TODO: Disallow !important - Use of !important. */
+  padding: 0 5px !important;
```

Comment:
There are a lot of `!important` here. It should be avoided as mentioned in `@TODO`.

If you for some reason think it is necessary then please add a comment explaining why.